insights/templates/homepage.vue.j2

```diff
@@ -45,11 +45,14 @@
 
   <main class="layout__content" >
     <div class="layout__content-inner">
-      <div class="intro-section">
-        <h2>Select a 360Giving dataset</h2>
-      </div>
-
       <div class="layout__header wrapper">
+
+        <div class="grid no-print">
```

Comment:
Despite this `no-print` the button appears on my print preview.

Reply:
Ah thanks I think I meant hide-print